deploy/operator/provision-shared-host-windows.yaml

```
@@ -0,0 +1,108 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: provision-shared-host-windows
+  namespace: multi-platform-controller
+spec:
+  description: >-
+    This task will retrieve SSH Private Key from a Windows host and create the relevant secret.
+  params:
+    - name: SECRET_NAME
+      type: string
+    - name: TASKRUN_NAME
+      type: string
+    - name: NAMESPACE
+      type: string
+    - name: HOST
+      type: string
+    - name: USER
+      type: string
+    - name: SUDO_COMMANDS
+      type: string
+  workspaces:
+    - name: ssh
+  stepTemplate:
+    env:
+      - name: SECRET_NAME
+        value: $(params.SECRET_NAME)
+      - name: TASKRUN_NAME
+        value: $(params.TASKRUN_NAME)
+      - name: NAMESPACE
+        value: $(params.NAMESPACE)
+      - name: HOST
+        value: $(params.HOST)
+      - name: USER
+        value: $(params.USER)
+      - name: SUDO_COMMANDS
+        value: $(params.SUDO_COMMANDS)
+  steps:
+    - name: provision
+      image: quay.io/redhat-appstudio/multi-platform-runner:01c7670e81d5120347cf0ad13372742489985e5f@sha256:246adeaaba600e207131d63a7f706cffdcdc37d8f600c56187123ec62823ff44
+      imagePullPolicy: IfNotPresent
+      volumeMounts:
+        - mountPath: /tls
+          name: tls
+      script: |
+        #!/bin/bash
+        cd /tmp
+        set -o verbose
+        set -eu
+        set -o pipefail
+        mkdir -p /root/.ssh
+        cp $(workspaces.ssh.path)/id_rsa /tmp/master_key
+        chmod 0400 /tmp/master_key
+        export SSH_HOST="${USER}@${HOST}"
+
+        export USERNAME=konflux-builder
+        ssh -i /tmp/master_key -o StrictHostKeyChecking=no "${SSH_HOST}" "powershell -Command cat C:\\Users\\Administrator\\${USERNAME}" > id_rsa
```

```suggestion
        ssh -i /tmp/master_key -o StrictHostKeyChecking=no "${SSH_HOST}" "powershell -Command cat C:\\Users\\${USERNAME}" > id_rsa
```
I think users are in C:\Users instead of C:\Users\Administrator